labs/lab4/chapter4.md

```diff
@@ -280,10 +261,10 @@ Which should output two pods, one called ```mariadb``` and one called
 already have it pulled up and verify the pods show up there as well.
 
 If you have any issues with the pods transistioning from a "Pending"
-state, you can check out the logs from the OpenShift service.
+state, you can check out the logs from the OpenShift containers.
 
 ```bash
-sudo journalctl -fl -u openshift
+minishift ssh "sudo journalctl -fl -u docker"
```

Comment:
`If you have any issues with the pods transistioning from a "Pending" state, you can check out the logs from the OpenShift containers` -- that's more about the infrastructure to bring up the container rather than the application isn't it? 

`oc describe pod` would be better IMHO